seed/analysis_pipelines/better.py

```diff
@@ -337,38 +359,36 @@ def _build_better_input(analysis_property_view, meters):
                                         )
                                     ),
                                     E.ResourceUses(
-                                        E.ResourceUse(
-                                            {'ID': 'Resource-' + str(11)},
-                                            E.EnergyResource('Electricity'),
-                                            E.ResourceUnits('kWh'),
-                                            E.EndUse('All end uses')
-                                        ),
-                                        E.ResourceUse(
-                                            {'ID': 'Resource-' + str(19)},
-                                            E.EnergyResource('Natural gas'),
-                                            E.ResourceUnits('MMBtu'),
-                                            E.EndUse('Heating')
-                                        ),
+                                        *[
+                                            E.ResourceUse(
+                                                {'ID': f'ResourceUse-{meter_idx:03}'},
+                                                E.EnergyResource(SEED_TO_BSYNC_RESOURCE_TYPE[meter.type]),
+                                                # SEED stores all meter readings as kBtu
+                                                E.ResourceUnits('kBtu'),
```

Comment:
Turns out seed stores all readings as kbtu, so no need to convert to anything else

Reply:
I think this should be fine just so long as ESPM does not have a meter type that is incompatible with `kBtu`.